modules/core/shared/src/main/scala/gem/math/Wavelength.scala

```diff
@@ -9,33 +9,97 @@ import gem.syntax.prism._
 import monocle.Prism
 
 /**
- * Exact wavelengths represented as unsigned integral angstroms in the range [0 .. Int.MaxValue]
- * which means the largest representable wavelength is 214.7483647 mm.
- * @param toAngstroms This wavelength in integral angstroms (10^-10 of a meter).
+ * Exact wavelengths represented as unsigned integral picometers in the range [0 .. Int.MaxValue]
+ * which means the largest representable wavelength is 2.147483647 mm.
```

Comment:
If this is problematic we could switch to `long` which would permit wavelengths of over 9 megameters.

Reply:
Is that necessary on the optical range?

Reply:
IR cuts off at 1 mm so this should be fine.